register/blake2/multihash_blake2.go

```diff
@@ -28,10 +28,20 @@ const (
 	blake2s_max = 0xb260
 )
 
+func stdBlake2FactoryMust(f func(key []byte) (hash.Hash, error)) func() hash.Hash {
```

Comment:
Nit: drop the blake2 here, this function is not doing anything blake-specific. Maybe `hasherMust`?

Reply:
Removing this function all together.